gems/sorbet-runtime/lib/types/private/methods/call_validation_2_6.rb

```diff
@@ -91,7 +99,6 @@ def self.create_validator_method_fast1(mod, original_method, method_sig, return_
       # This block is called for every `sig`. It's critical to keep it fast and
       # reduce number of allocations that happen here.
       # This method is a manually sped-up version of more general code in `validate_call`
-
```

Comment:
There are a handful of whitespace-only changes, feel free to view ignoring whitespace.